dnf-behave-tests/features/history-undo.feature

```diff
@@ -64,3 +64,25 @@ Scenario: Handle missing packages required for undoing the transaction
      And stderr contains "No package flac-1.3.2-8.fc29.x86_64 available."
      And stderr contains "No package wget-1.19.5-5.fc29.x86_64 available."
      And stderr contains "Error: no package matched"
+
+
+Scenario: Missing packages are skipped if --skip-unavailable is specified
+    When I execute dnf with args "install wget flac"
+    Then the exit code is 0
+     And Transaction is following
+         | Action        | Package                      |
+         | install       | wget-0:1.19.5-5.fc29.x86_64  |
+         | install       | flac-0:1.3.2-8.fc29.x86_64   |
+   When I drop repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-fedora-updates"
+   Then I execute dnf with args "update"
+    Then the exit code is 0
+     And Transaction is following
+         | Action        | Package                      |
+         | upgrade       | flac-0:1.3.3-3.fc29.x86_64   |
+         | upgrade       | wget-0:1.19.6-5.fc29.x86_64  |
+     Then I execute dnf with args "history undo last --skip-unavailable"
+     Then the exit code is 0
+     And Transaction is empty
+     And stderr contains "No package flac-1.3.2-8.fc29.x86_64 available."
+     And stderr contains "No package wget-1.19.5-5.fc29.x86_64 available."
```

Comment:
Please use `stderr is ...` with the exact output unless not possible for some reason?